src/main/java/com/example/parkingapp/api/rest/vm/LicensePlateVM.java

```diff
@@ -0,0 +1,45 @@
+package com.example.parkingapp.api.rest.vm;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+
+/**
+ * View Model to store car's license plate
+ */
+public class LicensePlateVM {
+    @NotNull
+    private String number;
+    @Pattern(regexp = "\\d{5}")
+    private String pinCode;
+
+    public LicensePlateVM() {
```

Comment:
czy ten publiczny konstruktor jest konieczny? czy musi być publiczny?

Reply:
Jedyne miejsce gdzie jest aktualnie wykorzystywany to test, więc można spokojnie zrefactorować i się go pozbyć